packages/flutter_tools/test/integration/test_data/stepping_project.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'project.dart';
+
+class SteppingProject extends Project {
```

Comment:
run `dartfmt` (as this is a new file)?

Reply:
Done! The only change was removing one blank line (which I guess is what you'd noticed). Thanks!